src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs

```diff
@@ -80,24 +96,43 @@ public void Criminal_compliance_check_returns_view(string gatewayPageId)
         [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.FailedToRepayFunds)]
         [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.ContractTermination)]
         [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.ContractWithdrawnEarly)]
-        public void Criminal_compliance_check_posted(string gatewayPageId)
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.RemovedRoTO)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.FundingRemoved)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.RemovedRegister)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.IttAccreditation)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.RemovedCharityRegister)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.Safeguarding)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.Whistleblowing)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.Insolvency)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.UnspentCriminalConvictions)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.FailedToRepayFunds)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.FraudIrregularities)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.OngoingInvestigation)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.ContractTerminated)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.WithdrawnFromContract)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.BreachedPayments)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.RegisterOfRemovedTrustees)]
+        [TestCase(GatewayPageIds.CriminalComplianceWhosInControlChecks.Bankrupt)]
+        public void Criminal_compliance_check_result_is_saved(string gatewayPageId)
         {
             var model = new OrganisationCriminalCompliancePageViewModel
             {
                 ApplicationId = Guid.NewGuid(),
                 ApplyLegalName = "legal name",
                 ComplianceCheckQuestionId = "CC-1",
                 ComplianceCheckAnswer = "No",
-                OptionPassText = null,
+                OptionPassText = "check passed",
                 Status = "Pass",
                 PageId = gatewayPageId,
                 QuestionText = "Question text",
-                Ukprn = "10001234"
+                Ukprn = "10001234",
             };
 
-            _applyApiClient.Setup(x => x.SubmitGatewayPageAnswer(It.IsAny<Guid>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>())).Returns(Task.CompletedTask);
+            _applyApiClient.Setup(x => x.SubmitGatewayPageAnswer(model.ApplicationId, gatewayPageId, model.Status,
```

Comment:
You don't really need the setup as you're not returning anything, the Verify will take care of it.